packages/react-components/src/organisms/ManuscriptVersionCard.tsx

```diff
@@ -269,11 +270,27 @@ const ManuscriptVersionCard: React.FC<ManuscriptVersionCardProps> = (
                         <Divider />
                       )}
                       <li css={additionalInformationEntryStyles}>
-                        <strong>Requesting APC Coverage?</strong>
+                        <strong>Requested APC Coverage?</strong>
                         <span css={additionalInformationValueStyles}>
                           {version.requestingApcCoverage}
                         </span>
                       </li>
+                      <li css={additionalInformationEntryStyles}>
```

Comment:
Should be conditional like the submissionDate
<img width="516" alt="Screenshot 2024-10-07 at 13 46 35" src="https://github.com/user-attachments/assets/926026a7-273f-4579-be71-36ccfd5fc83c">

Reply:
nice catch, thanks!